src/lib/rpc/svc_udp.c

```diff
@@ -272,12 +272,12 @@ static bool_t svcudp_reply(
      if (xdr_replymsg(xdrs, msg) &&
 	 (!has_args ||
 	  (SVCAUTH_WRAP(xprt->xp_auth, xdrs, xdr_results, xdr_location)))) {
-	  slen = (int)XDR_GETPOS(xdrs);
+	  slen = XDR_GETPOS(xdrs);
 	  if (sendto(xprt->xp_sock, rpc_buffer(xprt), slen, 0,
 		     (struct sockaddr *)&(xprt->xp_raddr), xprt->xp_addrlen)
 	      == slen) {
 	       stat = TRUE;
-	       if (su->su_cache && slen >= 0) {
```

Comment:
`slen` cannot be negative because `XDR_GETPOS()` returns `u_int`, and its value is not set elsewhere.